hedera-node/src/test/java/com/hedera/services/ledger/accounts/HashMapBackingTokens.java

```diff
@@ -0,0 +1,63 @@
+package com.hedera.services.ledger.accounts;
+
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.state.merkle.MerkleToken;
+import com.hederahashgraph.api.proto.java.TokenID;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class HashMapBackingTokens implements BackingStore<TokenID, MerkleToken> {
```

Comment:
Why do we need this class?

Reply:
It is used in `HederaLedgerLiveTest` and `LedgerBalanceChangesTest` to instantiate a `TransactionalLedger` for tokens